styles.css

```diff
@@ -5,12 +5,29 @@
   box-sizing: border-box;
 }
 
+html,
+body {
+  overflow-x: hidden;
+}
+
 #header {
   background-color: #3c3a39;
   height: 48px;
   padding: 12px;
 }
 
+.menu img {
+  width: 28px;
+}
+
+.desktop-nav-links header > span > img {
+  z-index: 1;
```

Comment:
- [ ] How about making your navigation mobile menu page a height of 100vh instead of 80vh? which will remove this bottom bararrow_down 👍🏻 

- [ ] Great job so far :raised_hands: but I think it would be better if you could use a `position: fixed;`  instead of absolute. Fixed position prevents your navigation page menu page from scrolling. The navigation is always fixed to the page instead of scrolling. Please change this. This will make your website similar to the original design. 💯 🥇 

- [ ] What about preventing the un-background from scrolling when the navbar is open? You can use a CSS property `overflow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 

- As requested by the previous reviewer, Your design doesn't quite match the template design. Some (but not all) of the things you should fix are background color, font, font size, margins, padding, and dividers. Please make sure your mobile menu looks identical to the Figma template.
- Figma design:

![Screenshot_20220511_052842](https://user-images.githubusercontent.com/25227022/167769517-0f2fb5ea-e6da-475b-891c-1978180c8547.png)

- Your design:

![fatigue](https://user-images.githubusercontent.com/79058364/173687367-590ef73a-11ff-4c5a-8626-7d170a2e4502.png)

-  Adding these features will make your design look good and look exactly like Figma's design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this